### PR TITLE
add movie-card

### PR DESCRIPTION
**Pull request by Saminsfw**

### CODE REVIEW

File: app/components/Rating/Rating.types.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+export type ratingType = {
```
Comment (UmmarHamid) [inline app/components/Rating/Rating.types.ts]:
```suggestion
export type RatingType = {
```

File: app/components/MenuPopup/MenuPopup.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    <Popover.Portal>
+      <Popover.Content
+        className="rounded-8 p-5 bg-white focus:bg-gray-50 border-2 border-violet-500"
+        sideOffset={5}
+      >
+        <div className="flex flex-col gap-2.5">
+          <label className="text-xs text-violet-500 ">Option 1</label>
+          <label className="text-xs text-violet-500 ">Option 2</label>
+          <label className="text-xs text-violet-500 ">Option 3</label>
+          <label className="text-xs text-violet-500 ">Option 4</label>
```
Comment (UmmarHamid) [inline app/components/MenuPopup/MenuPopup.tsx]: Can we use any tailwind color like lightBlue or green?

File: app/components/Rating/Rating.stories.ts [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  component: RatingComponent,
+  tags: ["autodocs"],
+  parameters: {
+    layout: "fullscreen",
+  },
+} satisfies Meta<typeof RatingComponent>;
+
+export default meta;
+type Story = StoryObj<typeof meta>;
+export const Hero_component: Story = {
```
Comment (UmmarHamid) [inline app/components/Rating/Rating.stories.ts]:
```suggestion
export const Rating_component: Story = {
```

File: app/components/MenuPopup/MenuPopup.types.ts [hunk 4]
```diff
@@ -0,0 +1,3 @@
+export type MenuPopupType = {
+  extraTriggerClasses?: string;
+};
```
Comment (UmmarHamid) [inline app/components/MenuPopup/MenuPopup.types.ts]:
```suggestion
export type MenuPopupType = {
 links: Array<LinkType>
  extraTriggerClasses?: string;
};
```

File: app/components/MenuPopup/MenuPopup.tsx [hunk 5]
```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import * as Popover from "@radix-ui/react-popover";
+import { DotsHorizontalIcon, Cross2Icon } from "@radix-ui/react-icons";
+import { MenuPopupType } from "./MenuPopup.types";
+
+const PopupMenu = ({ extraTriggerClasses }: MenuPopupType) => (
```
Comment (UmmarHamid) [inline app/components/MenuPopup/MenuPopup.tsx]:
```suggestion
const PopupMenu = ({ links, extraTriggerClasses }: MenuPopupType) => (
```

File: app/components/MenuPopup/MenuPopup.tsx (hunk 2 above)
Comment (UmmarHamid) [inline app/components/MenuPopup/MenuPopup.tsx]:
```suggestion
{links.map(link=><label className="text-xs text-violet-500 ">{link.text}</label>)}
```

File: app/components/MenuPopup/MenuPopup.tsx [hunk 6]
```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import * as Popover from "@radix-ui/react-popover";
+import { DotsHorizontalIcon, Cross2Icon } from "@radix-ui/react-icons";
```
Comment (UmmarHamid) [inline app/components/MenuPopup/MenuPopup.tsx]: use react-icons lib here and please remove the @radiix/reactui lib from the repo using yarn

**Review by UmmarHamid — CHANGES_REQUESTED** [review]
Looks good but some improvements can e done

File: app/components/MenuPopup/MenuPopup.tsx (hunk 6 above)
Comment (Saminsfw) [reply app/components/MenuPopup/MenuPopup.tsx]: Made the above mentioned changes and removed radix colors and icons as well @UmmarHamid

File: app/components/MenuPopup/MenuPopup.tsx [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+const PopupMenu = ({ links, extraTriggerClasses }: MenuPopupType) => (
+  <Popover.Root>
+    <Popover.Trigger asChild className={extraTriggerClasses}>
+      <button className="rounded-full w-[20px] h-[20px] inline-flex items-center justify-center bg-white opacity-60 hover:bg-lightGrey cursor-default outline-none">
+        <BiDotsHorizontalRounded />
+      </button>
+    </Popover.Trigger>
+    <Popover.Portal>
+      <Popover.Content
+        className="rounded-8 p-5 bg-gray-50 shadow-l border-2 border-lightBrown"
```
Comment (UmmarHamid) [inline app/components/MenuPopup/MenuPopup.tsx]:
```suggestion
        className="rounded-8 p-5 bg-gray-50 shadow-md border-2 border-lightBrown"
```

**Review by UmmarHamid — APPROVED** [review]

**Pull request merged**